### DISCUSSION TITLE
Import Figma screen as an image

### DISCUSSION THREAD

**Discussion by NitzanLaz** [simonoliver/UnityFigmaBridge#39] [Q&A]
Hey
I have a problem, hope you can help me with that. When I'm trying to import figma file , some of the screens are imported as an image instead of separated objects (Although I have buttons there). 
There is no difference between the screens that are imported good and the screens that don't.

**Comment by simonoliver** [comment]
Ah yep, this is probably becuase you have the frame marked as export. Try disabling export on the frame. I'll try and add this to the documentation

**Reply by NitzanLaz** [reply]
I tried it and it works, thank you very much!
another question, the animation I do in figma doesnt work in unity, are they supposed to be converted with your plugin?

**Reply by simonoliver** [reply]
The Figma REST API is currently missing (or was at least the last time I checked) the "transition" settings from prototype mode, so the plugin doesn't currently have the info needed to make this happen sadly.
